lib/std/zig/system/NativePaths.zig

```diff
@@ -11,6 +11,10 @@ const NativeTargetInfo = std.zig.system.NativeTargetInfo;
 include_dirs: ArrayList([:0]u8),
 lib_dirs: ArrayList([:0]u8),
 framework_dirs: ArrayList([:0]u8),
+
+nix_framework_dirs: ArrayList([:0]u8),
```

Comment:
I think we should figure out a way of satisfying nix-darwin without such special-casing like that.

Reply:
All nix users will have this problem. It's just that Darwin users also have an sdk root, so Zig attempts to add the paths in relation to, which is incorrect.